docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -35,17 +35,18 @@ Release status
 ``````````````
 This table links to the release notes for each major release. These release notes (changelogs) contain the dates and significant changes in each minor release.
 
-==============================      =================================================
-Ansible Release                     Status
-==============================      =================================================
-devel                               In development (2.10 unreleased, trunk)
-`2.9 Release Notes`_                Maintained (security **and** general bug fixes)
-`2.8 Release Notes`_                Maintained (security **and** critical bug fixes)
-`2.7 Release Notes`_                Maintained (security fixes)
-`2.6 Release Notes`_                Unmaintained (end of life)
-`2.5 Release Notes`_                Unmaintained (end of life)
-<2.5                                Unmaintained (end of life)
-==============================      =================================================
+==================================      =================================================
+Ansible Release                         Status
```

Comment:
Should this still mention Ansible? Now that it's base, we could either omit it or say Ansible Base instead. Alternatively, Maybe we need a separate table here?

Reply:
Option 1
```suggestion
Release                                 Status
```

Reply:
Option 2
```suggestion
Ansible Base Release                    Status
```